File: C++/A. Odd Divisor.cpp
#include <bits/stdc++.h>
using namespace std;
typedef long long lint;

int main(){
    int t,w,i;
    lint n;

    cin>>t;
    for(w=0;w<t;w++){
        cin>>n;
        double a;
        int b;
        
        if(n%2!=0)cout<<"YES\n";
        else{

            a = log2(n);
            b = a;
            if (a-b == 0)cout<<"NO\n";
            else cout<<"YES\n";
            
        }
    }

}

File: PYTHON/A. Odd Divisor.py
from math import log2
for w in range(int(input())):
    
    n = int(input())
    
    if (n%2 !=0):
        print('YES')
    else:
        a = log2(n)
        if a - int(a) == 0:
            print('NO')
        else:
            print('YES')
        